test/modules/aws-elb-integration.ts

```diff
@@ -193,6 +221,16 @@ describe('ELB Integration Testing', () => {
     WHERE load_balancer_name = '${lbName}';
   `, (res: any[]) => expect(res.length).toBe(0)));
 
+  it('deletes the security group 1', query(`
+    DELETE FROM security_group
+    WHERE group_name = '${sg1}';
+  `));
+
+  it('deletes the security group 2', query(`
+    DELETE FROM security_group
+    WHERE group_name = '${sg2}';
+  `));
```

Comment:
Nit: why not do these both in the same query? `WHERE group_name in ('${sg1}', '${sg2}')`